Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py

```diff
@@ -906,6 +910,17 @@ def setup_(self, step, stepName, stepDict, k, properties):
         # copy steps, then apply specializations
         UpgradeWorkflowPremix.setup_(self, step, stepName, stepDict, k, properties)
         UpgradeWorkflow_ProdLike.setup_(self, step, stepName, stepDict, k, properties)
+        #
+        if 'Digi' in step:
+            d = merge([stepDict[self.getStepName(step)][k]])
+            tmpsteps = []
+            for s in d["-s"].split(","):
+                if s == "DIGI:pdigi_valid" in s:
```

Comment:
I don't think this condition is correct

Reply:
Hmm.. Could you please guide? I get `-s DIGI,DATAMIX,L1TrackTrigger,L1,DIGI2RAW,HLT:@fake2` as expected.

Reply:
Either `if s == "DIGI:pdigi_valid"` or `if "DIGI:pdigi_valid" in s` would potentially make sense. Right now, this is comparing a string to a bool at some point.

Reply:
Thanks, Fixed.